contrib/babelfishpg_tsql/src/session.c

```diff
@@ -51,6 +53,18 @@ get_cur_db_name(void)
 	return pstrdup(current_db_name);
 }
 
+void 
+set_cur_db_name_for_parallel_worker(const char* logical_db_name)
+{
+	int len = strlen(logical_db_name);
+
+	Assert(len <= MAX_BBF_NAMEDATALEND);
```

Comment:
do we count null terminator part of MAX_BBF_NAMEDATALEND? MAX_BBF_NAMEDATALEND is max length or max available buffer?

Reply:
No, we assign one more extra bytes to store the NULL character.
```
static char current_db_name[MAX_BBF_NAMEDATALEND + 1] = {'\0'};
```
MAX_BBF_NAMEDATALEND is max available buffer